docs/fp_primer.md

```diff
@@ -258,8 +258,16 @@ digits of precision.
 
 For the CTS it is often sufficient to calculate the true value using TypeScript,
 since its native number format is higher precision (double-precision/f64), so
-all f64, f32, and f16 values can be represented in it. Where this breaks down
-will be discussed in the section on compile time vs runtime execution.
+all f64, f32, and f16 values can be represented in it. Unfortunately, the results of some operations cannot be exactly
```

Comment:
nit: Can you line wrap these to match the rest of the body text? (I think it is 80 column width)